easybuild/easyconfigs/n/ncbi-vdb/ncbi-vdb-2.11.2-gompi-2021b.eb

```diff
@@ -0,0 +1,57 @@
+easyblock = 'ConfigureMake'
+
+name = 'ncbi-vdb'
+version = '2.11.2'
+
+homepage = 'https://github.com/ncbi/ncbi-vdb'
+description = """The SRA Toolkit and SDK from NCBI is a collection of tools and libraries for
+ using data in the INSDC Sequence Read Archives."""
+
+toolchain = {'name': 'gompi', 'version': '2021b'}
+
+github_account = 'ncbi'
+source_urls = [GITHUB_SOURCE]
+sources = [{'download_filename': '%(version)s.tar.gz', 'filename': SOURCE_TAR_GZ}]
+patches = [
+    'ncbi-vdb-2.10.7_fix-LD_LIBRARY_PATH.patch',
+    'ncbi-vdb-cstdlib.patch',
+]
+checksums = [
+    '647efea2762d63dee6d3e462b1fed2ae6d0f2cf1adb0da583ac95f3ee073abdf',  # ncbi-vdb-2.11.2.tar.gz
+    'e8f22dbd0c2e564e296bafdf76ba0e0e2da0d13e22be5aaf322135e5f26eb133',  # ncbi-vdb-2.10.7_fix-LD_LIBRARY_PATH.patch
+    'db3d563262ca9b14e7b9a94a0be6683a9eef41a498c9c064aa05c2a6785f1655',  # ncbi-vdb-cstdlib.patch
+]
+
+builddependencies = [
+    ('Perl', '5.34.0', '-minimal'),
+    ('Python', '3.9.6', '-bare'),
+    ('CMake', '3.22.1'),
```

Comment:
@sassy-crick why are you adding CMake as a build dependency? This easyconfig uses the `ConfigureMake` easyblock, so it will not use cmake.

Reply:
thanks a lot for the explanation, since those are only build dependencies it is not very critical :+1: